dev_guide/deployments.adoc

```diff
@@ -109,6 +109,30 @@ created each time a new version of the `origin-ruby-sample:latest` image reposit
 <5> The link:#rolling-strategy[rolling strategy] is the default and may be omitted.
 ====
 
+== Deployment Client Tool Operations
+Use the 'ocs deploy' command with available options to start, view, cancel, or
+retry a deployment.
+
+----
+$ oc deploy <deployment_config> [--options]
+----
+
+[cols= 2]
+|===
+|`oc deploy <deployment_config> --latest`
+|Starts the specified deployment
+
+|`oc deploy <deployment_config>`
+|Dsiplays the specified deployment
+
+|`oc deploy <deployment_config> --retry`
+|Attempts to retry to deploy the last deployment. The deployer pod and any hook pods are deleted.
+
+|`oc deploy <deployment_config> --cancel`
+|Cancels the deployment that is currently in progress
```

Comment:
1) I think the "Deployment Client Tool Operations" section could be broken in disparate sections; maybe not the most perfect example, but consider the "Creating a `___`" / "Starting a `_____`" / "Viewing a `____`" style sections in these:

http://docs.openshift.org/latest/dev_guide/projects.html
http://docs.openshift.org/latest/dev_guide/builds.html

2) As for ordering, those ones are prrrrobably fine where they are? Seems like there's enough basic "what it is" info by that point (after "Overview" and "Creating a Deployment Configuration") to entertain starting a deployment, even if they made need to read further for more details on exactly how they want to achieve it.